server/channels/db/migrations/mysql/000116_create_oauth_outgoing_connection_table.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS OauthOutgoingConnection;
```

Comment:
Our tables are usually plural so this should probably be plural

```suggestion
DROP TABLE IF EXISTS OauthOutgoingConnections;
```